### ISSUE TITLE
Align and change color in Start Menu icon

### ISSUE THREAD

**Issue by diogoan** [Microsoft/vscode#58893]
I'm not sure if this is a bug or a feature request, but I thought I should bring up this here.

When fixing VScode shortcut in Start Menu (Windows 10), the icon doesn't seem to be correctly aligned. That can be seen comparing with Visual Studio 2017 icon, for example:

![image](https://user-images.githubusercontent.com/3026040/45702510-63ba6700-bb48-11e8-9b78-3675a9b7ecae.png)

Also, I have been using a blue tone in Windows theme for some time, when I installed/updated (don't remember exactly) VScode, which was using almost the same tone:

![image](https://user-images.githubusercontent.com/3026040/45702874-4508a000-bb49-11e8-9385-57419acf9d24.png)

A suggestion would be to use a theme-independent icon, likewise Visual Studio 2017 icon, at the left.

Windows color I was using:
![image](https://user-images.githubusercontent.com/3026040/45703016-96b12a80-bb49-11e8-8d24-5eaa6fff494d.png)

- VSCode version: 1.27.2
- Windows version: 1803

**Comment by octref** [comment]
/cc @joaomoreno since you are more familiar with Windows. I thought we have icons of the same size as VS, but why are they smaller and showing up so low to the frame?

> theme-independent icon, likewise Visual Studio 2017 icon, at the left.

If you make the tone purple, VS icon will be buried in the background color, correct? It seems there is no way to completely fix this.

**Comment by octref** [comment]
I see, maybe we didn't specify a bg color so the system falls back to the "tone" you are using, which might bury our icon.

**Comment by misolori** [comment]
@diogoan can you provide more info on how you're setting the tile colors and if you're using any windows customizations? I'm also using Win10 1803 and am not able to repro:

![image](https://user-images.githubusercontent.com/35271042/46308005-5603d900-c56d-11e8-86c1-71504cf359a7.png)

![image](https://user-images.githubusercontent.com/35271042/46307950-32d92980-c56d-11e8-91f6-53a342d2bbc3.png)

**Comment by diogoan** [comment]
@misolori I'm using vscode 1.27.2 on Win10 1803 too. I reinstalled vscode just to check how to reproduce and I'm not seeing the wrong icon/background again. Maybe that was corrected in a previous release?

![image](https://user-images.githubusercontent.com/3026040/46425931-6a81d600-c713-11e8-8905-2cc99e4bd72f.png)

I was using just an icon pack extension, but even installing it again, the icon is normal.
Thanks for the patience, if it happens again, I'll try to find what can be causing to bring here.
